### PR TITLE
Fix translations patch

### PR DESCRIPTION
**Pull request by tedw87**
Made change in the REST API for the PATCH method so it takes in consideration the translations.

### CODE REVIEW

**Comment by tedw87** [comment]
Tests are failing and I don't know why. I saw that the last commit before me, was failing with the same error.

**Comment by petschki** [comment]
Hi, the broken tests are unrelated and got just fixed on master branch. When master is green again, you can rebase your PR.

File: src/collective/taxonomy/restapi/services/taxonomy/update.py [hunk 1]
```diff
@@ -41,7 +44,8 @@ def generate_data_for_taxonomy(self, parsed_data, language, path=PATH_SEPARATOR)
             if subnodes:
                 new_path = "{}{}".format(new_path, PATH_SEPARATOR)
                 result.extend(
-                    self.generate_data_for_taxonomy(subnodes, language, new_path)
+                    self.generate_data_for_taxonomy(
```
Comment (petschki) [inline src/collective/taxonomy/restapi/services/taxonomy/update.py:47]: run `make style` to satisfy the linting check.

File: src/collective/taxonomy/restapi/services/taxonomy/update.py [hunk 2]
```diff
@@ -89,7 +93,8 @@ def reply(self):
                     taxonomy.data[language] = OOBTree()
 
             for language in taxonomy.data.keys():
-                data_for_taxonomy = self.generate_data_for_taxonomy(tree, language)
+                data_for_taxonomy = self.generate_data_for_taxonomy(
```
Comment (petschki) [inline src/collective/taxonomy/restapi/services/taxonomy/update.py:96]: see above

**Review by petschki — CHANGES_REQUESTED** [review]

**Comment by petschki** [comment]
Adding a note to `CHANGES.rst` helps keeping track of changes. We do not provide conventional changelogs in this package.

**Review by petschki — APPROVED** [review]

**Pull request merged**